deploy/kustomize/deploy_kustomize.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+set -x
+# Setup Environment Variables
+TARGET_NAMESPACE="${TARGET_NAMESPACE:-kubevirt-hyperconverged}"
+HCO_VERSION="${HCO_VERSION:-1.1.0}"
+HCO_CHANNEL="${HCO_CHANNEL:-1.1.0}"
+MARKETPLACE_MODE="${MARKETPLACE_MODE:-true}"
+HCO_REGISTRY_IMAGE="${HCO_REGISTRY_IMAGE:-quay.io/kubevirt/hco-container-registry:latest}"
+PRIVATE_REPO="${PRIVATE_REPO:-false}"
+QUAY_USERNAME="${QUAY_USERNAME:-}"
+QUAY_PASSWORD="${QUAY_PASSWORD:-}"
+QUAY_TOKEN=""
+CREATE_NAMESPACE="${CREATE_NAMESPACE:-false}"
+CONTENT_ONLY="${CONTENT_ONLY:-false}"
+KVM_EMULATION="${KVM_EMULATION:-false}"
+OC_TOOL="${OC_TOOL:-oc}"
+
+#####################
+
+SCRIPT_DIR="$(dirname "$0")"
+source $SCRIPT_DIR/get_quay_token.sh
```

Comment:
It is a good idea to just go ahead and define the function here.

Reply:
I moved `get_quay_token` and `retry_loop` functions to separate files mainly for readability, since in bash you must write the function definition before calling it. Unless we'll do something like [this](https://unix.stackexchange.com/questions/449498/call-function-declared-below), and we could put the auxiliary functions at the bottom of main script.

What do you think?